packages/react-devtools-extensions/src/__tests__/updateMockSourceMaps.js

```diff
@@ -52,8 +52,7 @@ function compile(fileName) {
   writeFileSync(
     resolve(externalDir, fileName),
     transformed.code +
-      `\n//# sourceMappingURL=${fileName}.map` +
-      `\n//# sourceURL=${fileName}`,
```

Comment:
@bvaughn Seems like we don't directly use this, and it was breaking the tests after I disabled Jest's stack trace translation in 556708cac7fae6e1667c82087c4b4f6a56369797. With this removed, the call stack contains the absolute path to the compiled file, which seems exactly right IMO.

Reply:
I'm not sure I understand. We use this for the external source map tests.

Reply:
I should have specified: we don't use `sourceURL`. We only use `sourceMappingURL`, which I kept.

Reply:
Oh, right. I added `sourceURL` b'c I thought it was typically present. You're saying this is what was causing the auto mapping behavior? :O

Reply:
The auto mapping was masking a different problem caused by `sourceURL`s in the test. The `fetch` mock started failing [here](https://github.com/facebook/react/blob/ed6c091fe961a3b95e956ebcefe8f152177b1fb7/packages/react-devtools-extensions/src/__tests__/parseHookNames-test.js#L53), because it started receiving the raw `sourceURL`s which couldn't be resolved from `__tests__` (now that they were no longer being replaced by source mapped absolute paths).

We *could* keep some form of `sourceURL` in the fixtures if there's something specific we want to test with it, in conjunction with beefing up the `fetch` mock again (which is super simple as of 8d8dd25163f83539e47765d99adc59f6731d695e). It could be a good way of testing the handling of absolute URLs, for example. But `sourceURL` and `sourceMappingURL` are distinct.